webrtc_to_hls/assets/src/ui.ts

```diff
@@ -1,30 +1,15 @@
-export function addVideoElement(
-  _: MediaStreamTrack,
-  stream: MediaStream,
-  mute: boolean = false
-) {
-  let video = <HTMLVideoElement>document.getElementById(stream.id);
+export function setPreview(stream: MediaStream) {
+  let video = <HTMLVideoElement | null>document.getElementById(stream.id);
+  if (video) return;
 
-  if (!video) {
-    video = document.createElement("video");
-    video.id = stream.id;
-    document.getElementById("videochat")?.appendChild(video);
-  }
+  video = document.createElement("video");
+  video.id = stream.id;
+
+  document.getElementById("preview")!.appendChild(video);
   video.srcObject = stream;
   video.autoplay = true;
   video.playsInline = true;
-  video.muted = mute;
-}
-
-export function removeVideoElement(_: MediaStreamTrack, stream: MediaStream) {
-  if (stream.getTracks().length > 0) {
-    return;
-  }
-
-  const video = <HTMLVideoElement>document.getElementById(stream.id);
-  if (video) {
-    video.remove();
-  }
+  video.muted = false;
```

Comment:
When running this app, I have audio feedback - is it caused, by setting this attribute to `false`?

Reply:
Do you mean you can hear your own audio with the streaming view?